### ISSUE TITLE
Verbose printing of guardian steps prints too many variable values

### ISSUE THREAD

**Issue by mmitche** [dotnet/arcade#14066]
- [ ] This issue is blocking 
- [ ] This issue is causing unreasonable pain

Variables that should typically not be printed are getting printed due to the way the SDL scripts are using Exec-BlockVerbosely: https://dev.azure.com/dnceng/internal/_build/results?buildId=2273416&view=logs&j=7d9eef18-6720-5c1f-4d30-89d7b76728e9&t=c5a86041-9185-53e8-42a2-1cadc4486f0d

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

**Comment by garath** [comment]
@mmitche to be clear, are you asking for dnceng to look into this? Or is this more of a note for yourself?

**Comment by riarenas** [comment]
This is for us

**Comment by mmitche** [comment]
Yeah, mainly for tracking. I was going to look into it if I found time but I didn't want it to get lost.

**Comment by missymessa** [comment]
Is this something we still want to do now that SDL stages have moved to the 1ES PTs?
